src/main/java/uk/gov/hmcts/darts/usermanagement/mapper/impl/UserAccountMapper.java

```diff
@@ -10,9 +10,6 @@
 import uk.gov.hmcts.darts.usermanagement.model.UserWithId;
 import uk.gov.hmcts.darts.usermanagement.model.UserWithIdAndLastLogin;
 
-import static uk.gov.hmcts.darts.common.enums.UserStateEnum.DISABLED;
-import static uk.gov.hmcts.darts.common.enums.UserStateEnum.ENABLED;
-
 @Mapper(componentModel = "spring",
```

Comment:
Are we mandating mapstruct now? For consistency 4 mappers use mapstruct out of a total of 18 mappers